index.md

```diff
@@ -108,7 +108,9 @@ Gravwell is pleased to announce our free [Community Edition](https://www.gravwel
 
     * [Labels and Filtering](gui/labels/labels.md)
 
-	* [Kits](kits/kits.md)
+    * [Kits](kits/kits.md)
+
+    * [Actionables](gui/actionables/actionables.md)
```

Comment:
I try to keep the index small by only having really big stuff on index.md, and I think actionables may be a bit advanced for it?